src/Azure.Functions.Cli/Diagnostics/LoggingFilterOptions.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Diagnostics;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace Azure.Functions.Cli
+{
+    public class LoggingFilterOptions
+    {
+        private const string DefaultLogLevelKey = "default";
+        private readonly string _hostJsonFileContent = string.Empty;
+
+        // CI EnvironmentSettings
+        // https://github.com/watson/ci-info/blob/master/index.js#L52-L59
+        public const string Ci = "CI"; // Travis CI, CircleCI, Cirrus CI, Gitlab CI, Appveyor, CodeShip, dsari
+        public const string Ci_Continuous_Integration = "CONTINUOUS_INTEGRATION";  // Travis CI, Cirrus CI
+        public const string Ci_Build_Number = "BUILD_NUMBER";  // Travis CI, Cirrus CI
+        public const string Ci_Run_Id = "RUN_ID"; // TaskCluster, dsari
+
+        public LoggingFilterOptions(bool verboseLogging = false)
+        {
+            VerboseLogging = verboseLogging;
+            if (IsCiEnvironment())
+            {
+                VerboseLogging = true;
```

Comment:
So there's no way to turn off verbose logging in CI environment? Can someone specify `--versboseLogging false` and have it force it to be false here?

Reply:
Good. Point. Added flag to figure out if Verbose flag is explicitly passed in command args